src/main/java/uk/ac/cam/cl/dtg/isaac/dao/GameboardPersistenceManager.java

```diff
@@ -537,7 +535,7 @@ public List<String> getInvalidQuestionIdsFromGameboard(final GameboardDTO gamebo
         gameboardDO.getContents().stream().map(GameboardContentDescriptor::getId).collect(Collectors.toList())));
 
     fieldsToMap.add(new GitContentManager.BooleanSearchClause(
-        TYPE_FIELDNAME, Constants.BooleanOperator.OR, Arrays.asList(QUESTION_TYPE, FAST_TRACK_QUESTION_TYPE)));
+        TYPE_FIELDNAME, Constants.BooleanOperator.OR, List.of(QUESTION_TYPE)));
```

Comment:
I could be wrong here, but doesn't this mean that instead of the `OR` operator, we now want `AND`? If they work the way I think, both of them should be equivalent in this case, but it feels strange to have an `OR` when there is only one possible value. I could be wrong, though. Shav had to deal with `BooleanOperator` when working on the random questions feature, so she may be able to help. If in doubt, and if it's hard to test, then we can always leave it as `OR`.